docs/devguides/yandex_dialogs.rst

```diff
@@ -0,0 +1,20 @@
+Yandex.Dialogs integration
```

Comment:
https://tech.yandex.ru/dialogs/
Yandex.Dialogs is a platform that also includes business chats and we only work with Alice.
Sorry for misguiding you on that before.

Reply:
I looked quickly at the documentation, and endpoints for business chats seem to have the same format...